### PR TITLE
Flow 0.142 (3/N): Update Animated Vendor

### PR DESCRIPTION
**Pull request by comp615**
Update Animated + EventEmmitter vendor files.

Tried to follow the same approach as previous vendor update #1955.
In this review, we assume part 2, and we update the vendored items. This was done by:
downloading react-native, and copy pasting the files to RNW.
Updating module.exports to export default
Updating Imports to reference RNW

Note that I slightly adjusted where the RNW Animated exports happened because React Native had a very similar thing, so I figured best to let it do generation of the component wrappers.

### CODE REVIEW

File: packages/react-native-web/src/vendor/react-native/Animated/AnimatedWeb.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
```
Comment (necolas) [inline packages/react-native-web/src/vendor/react-native/Animated/AnimatedWeb.js:11]: This file can be removed

**Review by necolas — COMMENTED** [review]
Is this taken straight from RN master branch or a release tag?

**Comment by comp615** [comment]
Straight from Master

**Comment by necolas** [comment]
Thanks. Merged

**Pull request closed without merging**